### ISSUE TITLE
[Visual Studio] Error: this template attempted to load component assembly 'Microsoft.VisualStudio.TemplateEngine.Wizard

### ISSUE THREAD

**Issue by DavidSimner** [Azure/Azure-Functions#301]
On:
- Windows 10.0 (Build 10240)
- Visual Studio version 15.3 (26510.0-Preview)
- Visual Studio 2017 Tools for Azure functions version 0.1 (released 2017-05-11 00:27:25)

I'm getting this error when I try to create an Azure Functions project:
![image](https://cloud.githubusercontent.com/assets/2973219/26255080/76880fb2-3cb0-11e7-86ec-398838af66d2.png)

```
Error: this template attempted to load component assembly
'Microsoft.VisualStudio.TemplateEngine.Wizard, Version=1.0.0.0,
Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. For more
information on this problem and how to enable this template, please
see documentation on Customizing Project Templates.
```

**Comment by phenning** [comment]
Which workloads were installed during the visual studio installation? Are you able to create any other ASP.NET Core templates?

**Comment by DavidSimner** [comment]
Installing the dependencies makes it work, so it might be worth adjusting the `.vsix` on your end so all the dependencies are listed in it, e.g. `Microsoft.VisualStudio.TemplateEngine`

**Comment by andrewchungxam** [comment]
^ in case it's helpful for others - I saw a similar error on my Visual Studio 2017 (Preview) install.

I had installed no other workloads since I just wanted to use the Azure Functions template.  I went back in and installed 2 additional workloads.
>1) ASP.NET and web development 2) Azure development

After this - it opened fine on VS 2017 (Preview)

^ By the way, to re-open the installer screen go to: File > New > Not finding what you're looking for? Open Visual Studio Installer
